### DISCUSSION TITLE
Adding Mobility Trends from Apple and Google

### DISCUSSION THREAD

**Discussion by ghost** [neherlab/covid19_scenarios#820] [Ideas]
Hi, would it be possible to add mobility data to the model? 

I found a study where they managed to input the mobility data from mobile phones into a SEIR model.

[Page e419] (https://www.researchgate.net/publication/343251460_Effects_of_human_mobility_restrictions_on_the_spread_of_COVID-19_in_Shenzhen_China_a_modelling_study_using_mobile_phone_data)

Some other studies also show a positive correlation between increase in mobility and increase in covid infections.

**Comment by TakumaKira** [comment]
Hi @pavimimos!

I tried adding the functionality [here](https://github.com/neherlab/covid19_scenarios/issues/715#issuecomment-689633219). Though I think the demo still may need some adjustments, what do you think of this?
